packages/is-prop-valid/src/props.js

```diff
@@ -69,6 +69,8 @@ const props = {
   draggable: true,
   encType: true,
   enterKeyHint: true,
+  fetchpriority: true,
```

Comment:
Is this even valid?

```suggestion
```

Reply:
It depends what you mean by valid, but I would say likely yes. I use this and it does indeed function and have an effect in browsers. The links I provided in the description also use it this way.

Reply:
I guess the handling of uppercase vs. lowercase confuses me a bit in `@emotion/is-prop-valid`. References:

- https://github.com/facebook/react/pull/25927
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65972

Maybe we should actually .toLowerCase() when determining if a prop should be forwarded or not.

Reply:
Ya it's possible converting to lower case could be useful. I didn't see many duplicates. For my use case with Styled Components, it currently passes without the capitol. So I think it would be needed for this PR to stop my warning. The camel case version is not one I've had to use with React yet.

Reply:
I think we should stick to the camelCase here and follow React's suite.

Reply:
The issue is that the camelCase only solution is not supported with the latest Next.js w/React. Only in React 19 will they be moving to camelCase for this property. If I change to using camelCase on elements within my codebase I get an error, as do others which is mentioned in the links that @oliviertassinari posted a year ago.

![image](https://github.com/user-attachments/assets/7a63bf2d-1541-4bbb-a3b8-532d56cd802c)